experimental/addons/envoy-gateway/README.md

```diff
@@ -0,0 +1,55 @@
+# envoy-gateway
+
+Envoy Gateway is an open source project for managing Envoy Proxy as a standalone or Kubernetes-based application gateway.
+
+## Installation
+
+Wait for Envoy Gateway to become available:
+
+```shell
+kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
+```
+
+Install the GatewayClass, Gateway, HTTPRoute and example app:
+
+```shell
+kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/latest/quickstart.yaml
+```
+
```

Comment:
Can you provide some samples about how KubeVela Application can use it?

I think we need to define some traits, right?

Reply:
This requires some refactors to gateway-trait and other gwapi implementations. Not sure if I should do whole things in a single PR or separated them.

Reply:
Sure